extensions/KHR/SPV_KHR_cooperative_matrix.asciidoc

```diff
@@ -0,0 +1,602 @@
+SPV_KHR_cooperative_matrix
+==========================
+
+Name Strings
+------------
+
+SPV_KHR_cooperative_matrix
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Jeff Bolz, NVIDIA
+- Daniel Koch, NVIDIA
+- Markus Tavenrath, NVIDIA
+- Kevin Petit, Arm Ltd.
+- Lei Zhang, Google
+- Ben Ashbaugh, Intel
+- Ruihao Zhang, Qualcomm
+- David Neto, Google
+- Tobias Hector, AMD
+- John Kessenich, Google
+- Nicolai Hahnle, AMD
+- Mariusz Merecki, Intel
+- Pedro Olsen Ferreira, Arm Ltd.
+- Ni Hui, Tencent
+- Dmitry Sidorov, Intel
+- Dong Wang, AMD
+- Ruimin Zhao, AMD
+- Alan Baker, Google
+- Lin Qun, AMD
+- Wooyoung Kim, Qualcomm
+- Krystian Andrzejewski, Intel
+
+Notice
+------
+
+Copyright (c) 2020-2023 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+
+- Approved by the SPIR-V Working Group: 2023-05-03
+- Not Yet Approved by the Khronos Board of Promoters: (To become approved, with an approval date, e.g. 2018-01-30)
```

Comment:
This should be updated with the ratification date.

Reply:
Good catch! I'd made the change locally (hence why it's in the HTML) but did not commit it.